### PR TITLE
Updated readme to include more details. 

### PR DESCRIPTION
**Pull request by PriyankaGitHub7**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,33 @@
 # GitStats
-A simple PreactJS app that uses the GitHub search API to display repositories in a list.
 
-## Running
-`npm install` then `npm run start`
+## Introduction
+A simple PreactJS app that uses the GitHub search API to display repositories in a list. 
+
+![alt text](https://github.com/PriyankaGitHub7/gitstats/blob/update-readme/src/assets/GitStatsView.png)
```
Comment (haroon-sheikh) [inline README.md]: @PriyankaGitHub7 can you replace PriyankaGitHub7 with ivanseed please.

**Comment by PriyankaGitHub7** [comment]
@haroon-sheikh Made the change as mentioned.

File: README.md [hunk 2]
```diff
@@ -1,8 +1,33 @@
 # GitStats
-A simple PreactJS app that uses the GitHub search API to display repositories in a list.
 
-## Running
-`npm install` then `npm run start`
+## Introduction
+A simple PreactJS app that uses the GitHub search API to display repositories in a list. 
+
+![alt text](https://github.com/ivanseed/gitstats/blob/update-readme/src/assets/GitStatsView.png)
```
Comment (haroon-sheikh) [inline README.md]: One more minor thing please... Can you please set `alt text` to be `gitstats.io`.

**Comment by PriyankaGitHub7** [comment]
@haroon-sheikh Done and resolved conflicts.

**Pull request merged**